packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -78,6 +80,7 @@ interface ImagePixelModule {
   pixelRepresentation: string;
   windowWidth: number;
   windowCenter: number;
+  voiLUTFunction: VOILUTFunctionType;
```

Comment:
can you please change this to be at least the same as the volume viewport? VOILUTFunction

Reply:
or this is different? i'm confused again :()

Reply:
I guess this is coming from cswil?

Reply:
This should be the same, I try to keep the same convention as the surrounding code. The `voiLUTFunction` is from cswil. Probably makes sense to change the other vars 

```
  pixelRepresentation => PixelRepresentation ;
  windowWidth: number; => I think this is derived dicom tag from voi
  windowCenter: number; =>  I think this is derived dicom tag from voi
```

Reply:
Ok good